lib/simpleofflinecache.js

```diff
@@ -18,25 +44,25 @@ SimpleOfflineCache.prototype.onFetch = function soc_onFetch(request, response) {
   }
 
   var clone = request.clone();
+  var self = this;
```

Comment:
nit: as we are now in a service worker environment where `self` refer to the ServiceWorkerGlobalScope, do you mind to change this by `that` or `_this` (I really like this last). wdyt?